src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -497,10 +506,10 @@ private function changeQuantityOfOrderItem(string $orderItemId, int $quantity, s
 
     private function hasItemWithNameAndQuantity(Response $response, string $productName, int $quantity): bool
     {
-        $items = json_decode($response->getContent(), true)['hydra:member'];
+        $items = $this->responseChecker->getCollection($response);
 
         foreach ($items as $item) {
-            if ($item['productName'] === $productName && $item['quantity'] === $quantity) {
+            if ($item['quantity'] === $quantity && $item['productName'] === $productName) {
```

Comment:
Why did you change the order?

Reply:
Comparing integer is faster 😄  but I can revert this change if it creates confusion 😉

Reply:
But there is a greater chance that quantity of different products could be the same and there will be more checks in the result